src/ui/default_input_event_bindings.ts

```diff
@@ -29,6 +29,7 @@ export function getDefaultGlobalBindings() {
     map.set("keyv", "toggle-default-annotations");
     map.set("keya", "toggle-axis-lines");
     map.set("keyo", "toggle-orthographic-projection");
+    map.set("keyt", "toggle-hide-cross-section-background-3d");
```

Comment:
I think for now we could avoid reserving a key for this behavior and just leave it in the settings.